src/axom/inlet/Table.hpp

```diff
@@ -1124,6 +1124,23 @@ class Table : public Verifiable<Table>
     return m_sidreGroup->hasView(detail::GENERIC_CONTAINER_FLAG);
   }
 
+  /*!
+   *****************************************************************************
+   * \brief Calls a function on the subtables corresponding to the elements
+   * of the container held by this table
+   * 
+   * \param [in] func The function to apply to individual container elements
+   * 
+   * \pre The calling table must be a generic container, i.e., isGenericContainer()
+   * returns true
+   * 
+   * \pre The function must accept a single argument of type Table&
+   * 
+   *****************************************************************************
+   */
+  template <typename Func>
+  void forEachContainerElement(Func&& func) const;
```

Comment:
Even though this is private, it seems odd to take the `func` parameter by r-value reference. I suppose it is easy enough to change later if you ever need to hang on to the functor for longer.

Reply:
In addition, this is actually a [universal reference](https://isocpp.org/blog/2012/11/universal-references-in-c11-scott-meyers), so it wouldn't preclude a `const` functor from being used with this function.